### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by LarryIVC**
### **In this PR, I did the following:**

- Create a new branch to develop the Works and About Myself sections.
- Added HTML layout for Works and About Myself sections.
- Added Styles to the Works and About Myself sections.
- Test project whit linter.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,21 +25,178 @@
...
+                    </li>
+                        <div class="lng-item">
+                            <img src="image/js-logo.png" alt="JS logo"><span>JavaScript</span>
+                        </div>
+                        <div class="lng-item">
+                            <img src="image/html-logo.png" alt="HTML logo"><span>HTML</span>
+                        </div>
+                        <div class="lng-item">
+                            <img src="image/css-logo.png" alt="CSS logo"><span>CSS</span>
+                        </div>
```
Comment (leonardodiasb) [inline index.html]: - [x] The list of languages works like a list and they would better fit inside the `li` of the languages. Every item inside a `ul` should be wrapped with the `li` tag for better semantics so please move the 'javascript', 'html' and 'css' items to inside the `li` of the languages above it.

File: styles.css [hunk 2]
```diff
@@ -60,21 +64,221 @@ main {
...
-.page1 ul li {
+.social li {
   margin-right: 20px;
 }
+
+.works {
+  width: 100%;
+  margin-top: 114px;
+  padding: 0 16px;
+}
```
Comment (leonardodiasb) [inline styles.css]: - [x] As the project required, it is mandatory to use flexbox and grid. For the main container of the works section, you should use css grid. Kindly use it to match the project requirements.

File: styles.css [hunk 3]
```diff
@@ -60,21 +64,221 @@ main {
...
+
+.works1 {
+  width: 100%;
+  background: white;
+  border-radius: 16px;
+  padding: 16px;
+  margin-bottom: 88px;
+  border: #dfe1e6 solid 1px;
+}
```
Comment (leonardodiasb) [inline styles.css]: - [x] As the project required, it is mandatory to use flexbox and grid. For each individual project of the works section, you should use CSS flexbox. Kindly use it to match the project requirements.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @LarryIVC, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Very professional README and PR message. ✔️
 - No linter errors. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (LarryIVC) [reply index.html]: I move the 'JavaScript', 'HTML' and 'CSS' items to inside the li of the languages above it.
image

File: styles.css (hunk 2 above)
Comment (LarryIVC) [reply styles.css]: @leonardodiasb, I insert `css grid` in the works section.
image

File: styles.css (hunk 3 above)
Comment (LarryIVC) [reply styles.css]: @leonardodiasb 
I adjust `Flexbox` in sections, Obs 3
image

**Review by divinecharlotte — APPROVED** [review]
# Hello @LarryIVC   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**